Artifacts/MsSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql

```diff
@@ -0,0 +1,33 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @applicationId INT;
```

Comment:
Should there be a corresponding script file for Postgres at _Artifacts/PgSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql_ ? I don't see it in the PR.

Reply:
I added Artifacts/PgSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql